src/gui/FileBrowser.cpp

```diff
@@ -454,179 +435,173 @@ void FileBrowserTreeWidget::keyPressEvent(QKeyEvent * ke )
 
 		contextMenu.addAction(
 			QIcon(embed::getIconPixmap("folder")),
 			tr("Open containing folder"),
-			[=]{ openContainingFolder(file); }
-		);
+			[=] { openContainingFolder(file); });
 
-		QAction* songEditorHeader = new QAction( tr("Song Editor"), nullptr );
+		QAction *songEditorHeader = new QAction(tr("Song Editor"), nullptr);
 		songEditorHeader->setDisabled(true);
-		contextMenu.addAction( songEditorHeader );
-		contextMenu.addActions( getContextActions(file, true) );
+		contextMenu.addAction(songEditorHeader);
+		contextMenu.addActions(getContextActions(file, true));
 
-		QAction* bbEditorHeader = new QAction( tr("BB Editor"), nullptr );
+		QAction *bbEditorHeader = new QAction(tr("BB Editor"), nullptr);
 		bbEditorHeader->setDisabled(true);
-		contextMenu.addAction( bbEditorHeader );
-		contextMenu.addActions( getContextActions(file, false) );
+		contextMenu.addAction(bbEditorHeader);
+		contextMenu.addActions(getContextActions(file, false));
 
 		// We should only show the menu if it contains items
-		if (!contextMenu.isEmpty()) { contextMenu.exec( e->globalPos() ); }
+		if (!contextMenu.isEmpty())
+		{
+			contextMenu.exec(e->globalPos());
+		}
 	}
 }
 
-
-
-
-QList<QAction*> FileBrowserTreeWidget::getContextActions(FileItem* file, bool songEditor)
+QList<QAction *> FileBrowserTreeWidget::getContextActions(FileItem *file, bool songEditor)
 {
-	QList<QAction*> result = QList<QAction*>();
+	QList<QAction *> result = QList<QAction *>();
 	const bool fileIsSample = file->type() == FileItem::SampleFile;
 
-	QString instrumentAction = fileIsSample ?
-		tr("Send to new AudioFileProcessor instance") :
-		tr("Send to new instrument track");
+	QString instrumentAction = fileIsSample ? tr("Send to new AudioFileProcessor instance") : tr("Send to new instrument track");
 	QString shortcutMod = songEditor ? "" : UI_CTRL_KEY + QString(" + ");
 
-	QAction* toInstrument = new QAction(
+	QAction *toInstrument = new QAction(
 		instrumentAction + tr(" (%2Enter)").arg(shortcutMod),
-		nullptr
-	);
+		nullptr);
 	connect(toInstrument, &QAction::triggered,
-		[=]{ openInNewInstrumentTrack(file, songEditor); });
+		[=] { openInNewInstrumentTrack(file, songEditor); });
 	result.append(toInstrument);
 
 	if (songEditor && fileIsSample)
 	{
-		QAction* toSampleTrack = new QAction(
+		QAction *toSampleTrack = new QAction(
 			tr("Send to new sample track (Shift + Enter)"),
-			nullptr
-		);
+			nullptr);
 		connect(toSampleTrack, &QAction::triggered,
-			[=]{ openInNewSampleTrack(file); });
+			[=] { openInNewSampleTrack(file); });
 		result.append(toSampleTrack);
 	}
 
 	return result;
 }
 
-
-
-
-void FileBrowserTreeWidget::mousePressEvent(QMouseEvent * me )
+void FileBrowserTreeWidget::mousePressEvent(QMouseEvent *me)
 {
 	// Forward the event
 	QTreeWidget::mousePressEvent(me);
 	// QTreeWidget handles right clicks for us, so we only care about left clicks
-	if(me->button() != Qt::LeftButton) { return; }
+	if (me->button() != Qt::LeftButton)
+	{
```

Comment:
I would prefer to have that "inline".

Reply:
IMO, this shouldn't have braces anyway if it's just returning. But I don't know what standard code style is here.

Reply:
We use explicit braces.

Reply:
Fixed using `AllowShortIfStatementsOnASingleLine: Always`